index.js

```diff
@@ -79,8 +80,11 @@ Electron.prototype._onfinish = function(){
 Electron.prototype._spawn = function(url){
   debug('spawn %s', url);
 
-  var self = this;
-  var ps = self.ps = spawn(electron, [runner], {
+  const self = this;
+  const args = [runner];
+  if (!this.sandbox) args.push('--no-sandbox');
```

Comment:
```suggestion
  if (!this.sandbox || process.getuid() === 0) args.push('--no-sandbox');
```
If process run from `root` (docker container for example) sandbox will not work and we need `--no-sandbox` argument. I'm not sure that this is the best places for this, but this need to be fixed :(

Reply:
I don't think it should default to `--no-sandbox` even with `root` user, in Docker or not, because that would automatically bypass the intended or recommended security defaults.  As @juliangruber suggested, the `--no-sandbox` option should be explicitly set and not assumed to be true for a `root` user, that could be very risky.

In addition, line 28 should be the place to put additional logic like this (and only for desirable changes). I think defaults and overrides for the same variable should be in the same section of the code or near each other, in order to avoid surprising behavior.